### ISSUE TITLE
Task :react-native-inappbrowser-reborn:compileDebugJavaWithJavac FAILED in v3.6.1

### ISSUE THREAD

**Issue by compwron** [proyecto26/react-native-inappbrowser#280]
### Make sure to check the demo app(s) for sample usage 
✅
### Make sure to check the existing issues in this repository 
✅
### If the demo apps cannot help and there is no issue for your problem, tell us about it
Please, ensure your title is less than 63 characters long and starts with a capital
letter.

### Which platform(s) does your issue occur on?
cannot build android react-native app with `yarn android`
Using SDK 29

### Please, provide the following version numbers that your issue occurs with:

- CLI: (run `react-native --version` to fetch it) 4.14.0
- Plugin(s): (look for the version numbers in the `package.json` file of your
project and paste your dependencies and devDependencies here)

No plugins
```
 "dependencies": {
    "@bugsnag/plugin-react-navigation": "^7.10.0",
    "@bugsnag/react-native": "^7.10.4",
    "@dudigital/react-native-zoomable-view": "^1.1.3",
    "@emotion/native": "^11.0.0",
    "@emotion/react": "^11.1.5",
    "@fortawesome/fontawesome-svg-core": "^1.2.30",
    "@fortawesome/free-regular-svg-icons": "^5.15.1",
    "@fortawesome/free-solid-svg-icons": "^5.14.0",
    "@fortawesome/react-native-fontawesome": "^0.2.5",
    "@react-native-community/masked-view": "^0.1.10",
    "@react-native-firebase/app": "^12.1.0",
    "@react-native-firebase/messaging": "^12.1.0",
    "@react-navigation/native": "^5.9.3",
    "@react-navigation/stack": "^5.14.2",
    "@styled-system/css": "^5.1.5",
    "@types/humps": "^2.0.0",
    "@types/jsdom": "^16.2.12",
    "@types/lodash.debounce": "^4.0.6",
    "@types/pluralize": "^0.0.29",
    "@types/react-native-background-timer": "^2.0.0",
    "@types/styled-system": "^5.1.10",
    "@types/styled-system__css": "^5.0.13",
    "axios": "^0.21.1",
    "base64url": "^3.0.1",
    "buffer": "^6.0.3",
    "corp-semantic-release": "^7.0.0",
    "eslint-plugin-react": "^7.24.0",
    "humps": "^2.0.1",
    "jest-mock-axios": "^4.3.0",
    "lodash": "^4.17.21",
    "pluralize": "^8.0.0",
    "prop-types": "^15.7.2",
    "react": "16.13.1",
    "react-native": "^0.63.3",
    "react-native-aes-crypto": "^1.3.9",
    "react-native-background-timer": "^2.4.1",
    "react-native-camera": "git+https://git@github.com/react-native-community/react-native-camera.git",
    "react-native-config": "^1.2.1",
    "react-native-contacts": "^6.0.5",
    "react-native-controlled-mentions": "^2.2.3",
    "react-native-device-info": "^8.1.3",
    "react-native-gesture-handler": "^1.10.2",
    "react-native-get-random-values": "^1.4.0",
    "react-native-image-pan-zoom": "^2.1.12",
    "react-native-inappbrowser-reborn": "^3.6.1",
    "react-native-keychain": "^7.0.0",
    "react-native-linear-gradient": "^2.5.6",
    "react-native-pager-view": "^5.2.1",
    "react-native-paper": "^4.9.2",
    "react-native-paper-tabs": "^0.4.2",
    "react-native-permissions": "^2.2.2",
    "react-native-reanimated": "^2.2.0",
    "react-native-safe-area-context": "^3.1.9",
    "react-native-screens": "^3.4.0",
    "react-native-splash-screen": "^3.2.0",
    "react-native-svg": "^12.1.0",
    "react-native-url-polyfill": "^1.1.2",
    "react-native-user-inactivity": "^1.1.0",
    "react-native-vector-icons": "^8.1.0",
    "react-native-webview": "^11.6.4",
    "styled-system": "^5.1.5",
    "url-search-params-polyfill": "^8.1.0"
  },
  "devDependencies": {
    "@babel/core": "^7.11.6",
    "@babel/preset-env": "^7.11.5",
    "@babel/preset-flow": "^7.10.4",
    "@babel/runtime": "^7.10.2",
    "@bugsnag/source-maps": "^2.0.0",
    "@emotion/babel-preset-css-prop": "^10.0.27",
    "@emotion/eslint-plugin": "^11.2.0",
    "@react-native-community/eslint-config": "^2.0.0",
    "@storybook/addon-actions": "^5.3",
    "@storybook/addon-knobs": "^5.3",
    "@storybook/addon-links": "^5.3",
    "@storybook/addon-ondevice-actions": "^5.3.23",
    "@storybook/addon-ondevice-knobs": "^5.3.25",
    "@storybook/react-native": "^5.3.25",
    "@storybook/react-native-server": "^5.3.23",
    "@testing-library/jest-native": "^3.4.2",
    "@testing-library/react-native": "7.1.0",
    "@types/humps": "^2.0.0",
    "@types/jest": "^26.0.0",
    "@types/lodash": "^4.14.170",
    "@types/react": "^16.9.36",
    "@types/react-native": "^0.63.35",
    "@types/react-test-renderer": "^16.9.2",
    "@typescript-eslint/eslint-plugin": "^3.2.0",
    "@typescript-eslint/parser": "^3.2.0",
    "babel-jest": "^26.3.0",
    "babel-loader": "^8.2.2",
    "babel-plugin-module-resolver": "^4.1.0",
    "babel-preset-react-native": "^4.0.1",
    "eslint": "^7.2.0",
    "eslint-import-resolver-babel-module": "^5.2.0",
    "eslint-plugin-import": "^2.22.1",
    "eslint-plugin-module-resolver": "^1.2.0",
    "eslint-plugin-react": "^7.24.0",
    "espree": "^7.3.0",
    "fetch-mock-jest": "^1.5.1",
    "jest": "^26.0.1",
    "jsdom": "^16.4.0",
    "metro-react-native-babel-preset": "^0.59.0",
    "react-dom": "16.13.1",
    "react-native-clean-project": "^3.4.0",
    "react-test-renderer": "16.11.0",
    "ts-jest": "^26.1.0",
    "typescript": "^3.9.5"
  },
  "jest": {
    "preset": "react-native",
    "setupFiles": [
      "./node_modules/react-native-device-info/jestSetup.js"
    ],
    "globals": {
      "window": {}
    }
  }
}
```

### Please, tell us how to recreate the issue in as much detail as possible. 
Upgrade to v3.6.1 (in our case, by dependabot PR)
Try `yarn android`

See error:
```

> Task :react-native-inappbrowser-reborn:compileDebugJavaWithJavac FAILED

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.4.1/userguide/command_line_interface.html#sec:command_line_warnings
271 actionable tasks: 2 executed, 269 up-to-date
/Users/me/ph/MyApp/node_modules/react-native-inappbrowser-reborn/android/src/main/java/com/proyecto26/inappbrowser/RNInAppBrowser.java:107: error: cannot find symbol
        builder.setNavigationBarColor(Color.parseColor(colorString));
               ^
  symbol:   method setNavigationBarColor(int)
  location: variable builder of type Builder
/Users/me/ph/MyApp/node_modules/react-native-inappbrowser-reborn/android/src/main/java/com/proyecto26/inappbrowser/RNInAppBrowser.java:116: error: cannot find symbol
        builder.setNavigationBarDividerColor(Color.parseColor(colorString));
               ^
  symbol:   method setNavigationBarDividerColor(int)
  location: variable builder of type Builder
2 errors

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-inappbrowser-reborn:compileDebugJavaWithJavac'.
> Compilation failed; see the compiler error output for details.
```

### Is there any code involved? 
 - provide a code example to recreate the problem 
 - (EVEN BETTER) provide a .zip with application or refer to a repository with application where the problem is reproducible.

Sorry, that's all I've got right now
![Screen Shot 2021-06-29 at 3 02 38 PM](https://user-images.githubusercontent.com/578159/123873266-179cd380-d8eb-11eb-83ce-9f7443084a92.png)

**Comment by jdnichollsc** [comment]
Duplicate of https://github.com/proyecto26/react-native-inappbrowser/issues/279

**Comment by jdnichollsc** [comment]
It was fixed in v3.6.2! <3
